tests/test_pyfunction.rs

```diff
@@ -164,30 +164,30 @@ fn test_function_with_custom_conversion_error() {
     );
 }
 
-#[test]
-fn test_raw_function() {
-    let gil = Python::acquire_gil();
-    let py = gil.python();
-    let raw_func = raw_pycfunction!(optional_bool);
-    let fun = PyCFunction::new_with_keywords(raw_func, "fun", "", py.into()).unwrap();
-    let res = fun.call((), None).unwrap().extract::<&str>().unwrap();
-    assert_eq!(res, "Some(true)");
-    let res = fun.call((false,), None).unwrap().extract::<&str>().unwrap();
-    assert_eq!(res, "Some(false)");
-    let no_module = fun.getattr("__module__").unwrap().is_none();
-    assert!(no_module);
-
-    let module = PyModule::new(py, "cool_module").unwrap();
-    module.add_function(fun).unwrap();
-    let res = module
-        .getattr("fun")
-        .unwrap()
-        .call((), None)
-        .unwrap()
-        .extract::<&str>()
-        .unwrap();
-    assert_eq!(res, "Some(true)");
-}
+// #[test]
+// fn test_raw_function() {
+//     let gil = Python::acquire_gil();
+//     let py = gil.python();
+//     let raw_func = raw_pycfunction!(optional_bool);
+//     let fun = PyCFunction::new_with_keywords(raw_func, "fun", "", py.into()).unwrap();
+//     let res = fun.call((), None).unwrap().extract::<&str>().unwrap();
+//     assert_eq!(res, "Some(true)");
+//     let res = fun.call((false,), None).unwrap().extract::<&str>().unwrap();
+//     assert_eq!(res, "Some(false)");
+//     let no_module = fun.getattr("__module__").unwrap().is_none();
+//     assert!(no_module);
+
+//     let module = PyModule::new(py, "cool_module").unwrap();
+//     module.add_function(fun).unwrap();
+//     let res = module
+//         .getattr("fun")
+//         .unwrap()
+//         .call((), None)
+//         .unwrap()
+//         .extract::<&str>()
+//         .unwrap();
+//     assert_eq!(res, "Some(true)");
+// }
```

Comment:
I think let's remove this test and the accompanying `raw_pycfunction` macro. If users ask for something to be restored we could look at making a new API that suits the possibility of different C functions.

Reply:
Will do.